spec/datadog/core/environment/variable_helpers_spec.rb

```diff
@@ -201,11 +201,45 @@
         it { is_expected.to eq(%w[1 2]) }
       end
 
-      context ' 1 , 2 , 3 ' do
-        let(:env_value) { ' 1 , 2 , 3 ' }
+      context ' 1 , 2 ,  3 ' do
+        let(:env_value) { ' 1 , 2 ,  3 ' }
+
+        it { is_expected.to eq(%w[1 2 3]) }
+      end
+
+      context '1 2 3' do
+        let(:env_value) { '1 2 3' }
 
         it { is_expected.to eq(%w[1 2 3]) }
       end
+
+      context '1,2 3' do
+        let(:env_value) { '1,2 3' }
+
+        it { is_expected.to eq(['1', '2 3']) }
+      end
```

Comment:
To be honest, I'm not sure how useful it is to support this, rather than just doing split by space and comma at the same time and be done with it. But it is true that the Python implementation also does some of these hijinks so...

(To be honest, we're not matching the [exact hijinks](https://github.com/DataDog/dd-trace-py/pull/2196/files) with spaces in python, e.g. `(" key: val,key2:val2", {" key": " val", "key2": "val2"}, None),`, but again, I'd just go for splitting on both at the same time and not supporting weird mixes).

Reply:
I'm following [Go's implementation](https://github.com/DataDog/dd-trace-go/pull/837/files#diff-0d5b206a111ef966111c0ebc8ee49a5557dac8d91e8d649dcb96e71df51f7ef0R142), [Java's implementation](https://github.com/DataDog/dd-trace-java/pull/2011/files#diff-89d6b8e798c61c51db0468581d610d0854ecdfb537085667227dacc1880825a1R72-R78), and Python's original implementation document (internal doc):
* Looking for commas, and using comma-separated if so.

I'm trying to match their implementations, to keep consistency, instead of doing what seems more right here.